docs/source/contributing/translation.rst

```diff
@@ -0,0 +1,51 @@
+====================
+Internationalization
+====================
+
+If you want to help with translating the docs:
+
+#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows Manim team to review and help to improve suggestions from the community. You will use Transifex to submit your translation suggestions.
+#. Open to the `Manim project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
+#. Click on Join Team.
+#. Select the language you want to help with.
+
+Congratulations ! You are now a Translator for the Manim Community project!
+
+
+You have access to the projects you joined just on the left of your profile picture
+on the top bar.
+
+.. image:: /_static/select_transifex_repo.png
+   :align: center
+   :width: 200
+
+Then you can go the
+`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_
+and select the language you want to help with. Then you have a blue button reading
+`Translate`, click it to get started with translating ! You will get to a list of
+all resources of the project. You can click on any untranslated resource, then
+click on `Translate` and you will get the following page:
+
+.. image:: /_static/transifex_explanation.png
+    :align: center
+
+The `A` panel is the list of all the strings in the resource. Click on any of
+them to translate it. It will appear in the `B` panel, and you will be able to
+set your translation in the `C` panel. Once you're done, click `Save changes`.
+Your translation will have to be reviewed before getting in the docs.
+You can also get back to the resources list by clicking `d` or change the
+destination language in `e`. Your
+
+
+If you want to become a reviewer, please join the discord server of Manim and
+you can ping the role @i18nService.
+
+If you want to add a new language, you can go to
+`the project page <https://www.transifex.com/manim-community/manim-community-documentation>`_
+If you didn't joined the project yet, you will have a link which reads
+`Request language`, and send the request.
+If you are already in the project, go to the languages page and you have a
+blue button reading `Request language` and do your request.
```

Comment:
```suggestion
```

Reply:
That edit changes it to:
If you are already in the project, go to the languages page and you have a
...
...

Reply:
No, combined with the suggestion on the previous line it should be correct.

Reply:
Ohh I see. To be honest, those suggestions should have been combined.

Reply:
Ah, I didn't know we could do multiple line suggestions, but I do now. Thanks!